app/res/layout/entity_map_view.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<fragment xmlns:android="http://schemas.android.com/apk/res/android"
+    android:name="com.google.android.gms.maps.MapFragment"
```

Comment:
Is there a reason you didn't use `com.google.android.gms.maps.SupportMapFragment` here instead as suggested here https://developers.google.com/maps/documentation/android-api/start ?